0miner

```diff
@@ -260,7 +260,7 @@ then
   OPTS="$(dgh_enabled_devices '--devices ' ',' 'N' "$OPTS")"
   UCCALGO="-a ${ALGO,,}"
   HCD="${NVOC}"/miners/"${!xminer}"/"${!xversion}"/ccminer
-  eval $LAUNCH $HCD $UCCALGO -o ${!xproto}://"${!xpool}":"${!xport}" -u "${!xaddr}""${!xwallet}""${!xwork}" -p "$MINER_PWD" -i "${!xintensity}" $OPTS
+  eval $LAUNCH $HCD $UCCALGO -o ${!xproto}://"${!xpool}":"${!xport}" -u "${!xaddr}""${!xwallet}""${!xwork}" -p "$MINER_PWD" -i "${!xintensity}" $OPTS $CCMINER_OPTS
```

Comment:
CCMINER_OPTS has to be deprecated and removed as well. If someone still likes to create a common CCMINER_OPTS variable this should be done in 1bash, then set every TPccminer_OPTS, XXccminer_OPTS to $CCMINER_OPTS if desired.